### PR TITLE
Unit testing task

### PR DESCRIPTION
**Pull request by aliaksandr-harunou**

### CODE REVIEW

File: test/specs/add.spec.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+describe(`positive tests for addition`, () => {
+  let calc;
+  beforeEach(() => {
+    calc = new Calculator();
+  });
+  afterEach(() => {
+    calc = null;
+  });
+
+  it(`shoud return summ of positive integer numbers`, () => {
```
Comment (VladSelikh) [inline test/specs/add.spec.js]: Please correct a misprint: **should**, **sum**. The same is for other tests.

File: test/specs/multiply.spec.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  });
+  afterEach(() => {
+    calc = null;
+  });
+
+  it(`shoud return summ of positive integer numbers`, () => {
+    expect(calc.multiply(4, 3, 6)).to.be.equal(4*3*6);
+  });
+
+  it(`shoud return summ of negative integer numbers`, () => {
```
Comment (VladSelikh) [inline test/specs/multiply.spec.js]: Does the `multiply()` method return a sum?

File: test/specs/add.spec.js (hunk 1 above)
Comment (aliaksandr-harunou) [reply test/specs/add.spec.js]: fixed

File: test/specs/multiply.spec.js (hunk 2 above)
Comment (aliaksandr-harunou) [reply test/specs/multiply.spec.js]: fixed

**Review by VladSelikh — APPROVED** [review]

**Pull request merged**